UnitedIncome/components
# Tile Border Color (isDark)

### Description
> Please provide a brief description of your changes below.

Adjusts the border color for the `isDark` variant of the `Tile` so it's at 30% opacity. Spoke to Monique about this last night before leaving for the day. 

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Tile should be the same, except in dark mode where the top color is now slightly more opaque.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

<img width="1452" alt="Screen Shot 2020-05-05 at 5 29 29 AM" src="https://user-images.githubusercontent.com/10888441/81052430-6985cd00-8e91-11ea-8735-548ddda20533.png">

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
